tests/zkasm/assert.js

```diff
@@ -0,0 +1,68 @@
+const fs = require("fs");
+
+class InstrumentInst {
+    /**
+     * @param {Object} config 
+     * @param {string} [config.outputFile] - the file to which output will be written
+     */
+    constructor(config) {
+        this.config = config;
+        this.results = {};
+    }
+
+    setup() {
+        // `zkevm-proverjs` requires a `setup` function on helper objects.
+    }
+
+    /**
+     * 
+     * @param {Object} ctx - context of zkasm program
+     * @param {Object} tag - information of helper function. 
+     * Helper function must be used in format:
+     * ${assert_eq(regname, regname, tag)}, for example:
+     * ${assert_eq(A, B, test_generated_from_line_3000)}
+     */
+    eval_assert_eq(ctx, tag) {
+        // TODO print verbose error on invalid params
+        // TODO: why arg* is list of two integers with last zero and first value in register???
+        // Is it chunks? If yes they should be merged instead of ignoring second one
```

Comment:
I feel like this TODO should be resolved before merging. A starting point to find out more about `ctx` and `tag` passed to helpers might be [helper invocation](https://github.com/0xPolygonHermez/zkevm-proverjs/blob/f5922ccc197318159b38aace6df9f0467e478205/src/sm/sm_main/sm_main_exec.js#L3086) in `zkevm-proverjs`.